### PR TITLE
refactor: remove old tests files which have been merged

### PR DESCRIPTION
**Pull request by saidsay-so**

### CODE REVIEW

File: rust/src/tests/chflags.rs [hunk 1]
```diff
@@ -75,6 +75,7 @@ fn get_flags(ctx: &TestContext) -> (FileFlag, FileFlag, FileFlag) {
 
 crate::test_case! {
     /// chflags(2) set the flags provided for the file.
+    // chflags/00.t
```
Comment (asomers) [inline rust/src/tests/chflags.rs:78]: I actually think it's still useful to leave these comments in place.

**Review by asomers — CHANGES_REQUESTED** [review]
In reviewing my own past PRs, I noticed that I actually left out a few things from the granular tests.  I have a PR in progress to fix that.  So please don't delete them yet. Did you review the other changes, too?

**Comment by musikid** [comment]
> In reviewing my own past PRs, I noticed that I actually left out a few things from the granular tests. I have a PR in progress to fix that. So please don't delete them yet. Did you review the other changes, too?

If you're talking about your tests other than the granular ones, then yes I did a quick review of those.

File: rust/src/tests/chflags.rs (hunk 1 above)
Comment (musikid) [reply rust/src/tests/chflags.rs:78]: I agree, that's why I added those which were missing.

**Review by asomers — CHANGES_REQUESTED** [review]
But for that one missing comment, it LGTM

**Pull request merged**